Misc/NEWS.d/next/Library/2023-06-25-16-54-24.gh-issue-106078.cFnYUM.rst

```diff
@@ -0,0 +1 @@
+Convert :mod:`_decimal` types to heap types.
```

Comment:
Maybe we can skip news for individual PRs and add one in the end?

Reply:
Yes. I agree with you. I checked the PRs related to isolate `_io` and that's what they do.

Reply:
> Maybe we can skip news for individual PRs and add one in the end?

Yes, that's been the practice so far.